packages/libs/react-components/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,77 @@
+import { SystemIcons } from '../Icons';
+
+import {
+  AbsoluteButton,
+  StyledButton,
+  StyledHeading,
+  StyledIconButton,
+  StyledNotification,
+  StyledText,
+} from './styles';
+
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcons)[keyof typeof SystemIcons];
+
+  title: string;
+  description?: string;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+
+  actionButtonLabel?: string;
+  actionButtonOnClick?: () => void;
+  displayActionButton?: boolean;
+
+  color?: string;
```

Comment:
All variants should be typed with stitches variant type: https://stitches.dev/docs/typescript#extract-variants-from-a-component